v2/spanner-common/src/test/java/com/google/cloud/teleport/v2/spanner/migrations/metadata/CassandraSourceMetadataTest.java

```diff
@@ -38,24 +41,24 @@ class CassandraSourceMetadataTest {
 
   private CassandraSourceMetadata.Builder builder;
 
-  @BeforeEach
-  void setUp() {
+  @Before
+  public void setUp() {
     MockitoAnnotations.openMocks(this);
```

Comment:
Could we consider using
`@RunWith(MockitoJUnitRunner.class)` instead of `@RunWith(JUnit4.class)` especially for classes that use `@Mock` ? 
1. `MockitoJUnitRunner` extends Junit Runner , 
2. It automatically does `MockitoAnnotations.openMocks(this);` , so we won't need this is we  `RunWith` `MockitoJUnitRunner.class`
3. Most importantly, it also validates mockito framework usage after each test method. - Which is a good plush.